gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcWriteChannel.java

```diff
@@ -64,12 +69,20 @@
   // Default GCS upload granularity.
   static final int GCS_MINIMUM_CHUNK_SIZE = 256 * 1024;
 
-  private static final Duration START_RESUMABLE_WRITE_TIMEOUT = Duration.ofMinutes(10);
-  private static final Duration QUERY_WRITE_STATUS_TIMEOUT = Duration.ofMinutes(10);
-  private static final Duration WRITE_STREAM_TIMEOUT = Duration.ofMinutes(20);
+  private static final Duration START_RESUMABLE_WRITE_TIMEOUT = Duration.ofSeconds(10);
+  private static final Duration QUERY_WRITE_STATUS_TIMEOUT = Duration.ofSeconds(10);
```

Comment:
Isn't this too low? Can we make is larger just to make sure that we are resilient to network issues? Maybe at least 1 minute.

Reply:
Updated. Although this is supposely not kind of requests that should take a minute before triggering retry.

Reply:
It shouldn't take a minute, but 10 seconds was kind of short. You could use 30 seconds or just leave it at 1 min.